### ISSUE TITLE
Unable to install the gem

### ISSUE THREAD

**Issue by gouravmodi** [Shopify/active_shipping#258]
![screenshot from 2015-05-08 15 15 44](https://cloud.githubusercontent.com/assets/9841879/7534120/22900cd4-f595-11e4-8a46-106790a1dcb5.png)

**Comment by wvanbergen** [comment]
What version of Ruby/Rubygems are you using?

**Comment by gouravmodi** [comment]
i am using ruby 2.1.2

**Comment by wvanbergen** [comment]
It installs fine for me on 2.1:

```
$ gem install active_shipping
Fetching: active_utils-3.0.0.gem (100%)
Successfully installed active_utils-3.0.0
Fetching: quantified-1.0.1.gem (100%)
Successfully installed quantified-1.0.1
Fetching: active_shipping-1.0.0.gem (100%)
Successfully installed active_shipping-1.0.0
3 gems installed
```

In your case it actually looks like it installed it, but something went wrong post install. Without something like a stack trace, it's hard to figure out what though.

**Comment by gouravmodi** [comment]
Successfully installed active_shipping-1.0.0
Exception `Gem::LoadError' at /home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency.rb:298 - Could not find 'rdoc' (~> 4.0.0) - did find: [rdoc-4.2.0,rdoc-4.1.2,rdoc-4.1.0,rdoc-3.12.2]
Using Ext extension for JSON.
Exception `TypeError' at /home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:450 - no implicit conversion of Array into String
ERROR:  While executing gem ... (TypeError)
    no implicit conversion of Array into String
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:450:in `exist?'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:450:in `block in check_files'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:449:in `delete_if'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:449:in `check_files'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:525:in `finish'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/options.rb:1123:in `parse'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:177:in `block in generate'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:174:in `chdir'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:174:in `generate'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:56:in `block in generation_hook'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:55:in `each'
	/home/gouravm/.rvm/gems/ruby-2.1.2/gems/rdoc-4.2.0/lib/rdoc/rubygems_hook.rb:55:in `generation_hook'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:400:in `call'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:400:in `block (2 levels) in install'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:399:in `each'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:399:in `block in install'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:353:in `in_background'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/dependency_installer.rb:398:in `install'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/commands/install_command.rb:219:in `install_gem'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/commands/install_command.rb:263:in `block in install_gems'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/commands/install_command.rb:259:in `each'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/commands/install_command.rb:259:in `install_gems'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/commands/install_command.rb:171:in `execute'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/command.rb:305:in `invoke_with_build_args'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/command_manager.rb:167:in `process_args'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/command_manager.rb:137:in `run'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/rubygems/gem_runner.rb:54:in `run'
	/home/gouravm/.rvm/rubies/ruby-2.1.2/bin/gem:21:in `<main>'



Here is the stack trace

**Comment by wvanbergen** [comment]
Looks like something is wrong with rdoc. You can skip it using `gem install active_shipping --no-rdoc`

**Comment by wvanbergen** [comment]
Fixed in version 1.0.1
